docs/update_examples.py

```
@@ -67,7 +67,27 @@
 
             reporthook.pbar = None
 
-            urllib.request.urlretrieve(url, local_filepath, reporthook)
+            # Try with default SSL context first
+            try:
+                urllib.request.urlretrieve(url, local_filepath, reporthook)
+            except urllib.error.URLError as ssl_err:
+                if "CERTIFICATE_VERIFY_FAILED" in str(ssl_err) or "certificate verify failed" in str(ssl_err):
+                    print(f"⚠️ SSL certificate verification failed, trying with SSL context that bypasses verification...")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)